src/game.cpp

```diff
@@ -5865,9 +5871,24 @@ int main(int argc, char** argv)
 					{
 						continue;
 					}
-					if ((subwindow && !players[i]->shootmode) || (gamePaused && i == clientnum))
+
+					if ( gamePaused )
```

Comment:
this block doesn't compile in editor, add `#ifndef EDITOR` here

Reply:
Don't believe game.cpp is linked by editor, so no issues here